migrations/V36__Add_Plan_Document_Solution_Link.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE plan_document_solution_link (
+    id UUID PRIMARY KEY NOT NULL,
+    model_plan_id UUID NOT NULL,
```

Comment:
`model_plan_id` is a duplicate key here, I don't think it should be in this table as it is already in the plan_document table. Solution to Document is really the only link we need, we can query other data through joins.